swap_meet/item.py

```diff
@@ -1,2 +1,38 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+
+        #refactor validation of condition
+        self.condition = condition
+        if 0 <= self.condition <= 5:
+            self.condition = float(condition)
+        else:
+            raise ValueError("Condition must range from 0-5.")
```

Comment:
If you're going to raise an error, I'd do it as early as possible, to avoid executing unnecessary code :)